src/Symfony/Component/Console/Application.php

```diff
@@ -1137,6 +1139,7 @@ protected function getDefaultInputDefinition(): InputDefinition
             new InputOption('--version', '-V', InputOption::VALUE_NONE, 'Display this application version'),
             new InputOption('--ansi', '', InputOption::VALUE_NEGATABLE, 'Force (or disable --no-ansi) ANSI output', null),
             new InputOption('--no-interaction', '-n', InputOption::VALUE_NONE, 'Do not ask any interactive question'),
+            new InputOption('--no-progress', null, InputOption::VALUE_NONE, 'Disable the progress bar output during the execution of the command'),
```

Comment:
Composer also has a `--no-progress` option for this ([see in compose source](https://github.com/search?q=repo%3Acomposer%2Fcomposer%20no-progress&type=code)). This confirms that it's the appropriate name.

But this is going to create a conflict, and all the commands that define this option will fail if the option is added globally.
https://github.com/symfony/symfony/blob/6ef1ab4e46d7fe51b60a71229537a7bec1543a7e/src/Symfony/Component/Console/Input/InputDefinition.php#L219-L221

Reply:
Oh, I misunderstood what the hidden options were. There really is a duplicate in the new `OutputInterface` methods since you give 2 ways to configure and retrieve the same thing.